js/app/utils.js

```diff
@@ -451,6 +451,19 @@ function record_search_metric (query) {
         [query, app_id]));
 }
 
+const CONTENT_ACCESS_METRIC_EVENT_ID = 'fae00ef3-aad7-44ca-aff2-16555e45f0d9';
+// Should be mocked out during tests so that we don't actually send metrics
+function record_content_access_metric (open, ekn_id, title, content_type) {
+    let app_id = Gio.Application.get_default().application_id;
+    let recorder = EosMetrics.EventRecorder.get_default();
+    if (open)
+        recorder.record_start(CONTENT_ACCESS_METRIC_EVENT_ID, new GLib.Variant('s', ekn_id),
+            new GLib.Variant('(sss)', [app_id, title, content_type]));
+    else
+        recorder.record_stop(CONTENT_ACCESS_METRIC_EVENT_ID, new GLib.Variant('s', ekn_id),
+            new GLib.Variant('(sss)', [app_id, title, content_type]));
```

Comment:
You don't actually have to send the payload along with the stop metric a second time, only the key. So the second variant can be empty.